ui/pages/Project/components/ProjectOverview.jsx

```diff
@@ -154,91 +213,85 @@ const ProjectOverview = React.memo((
       key: 'blank' })
   }
 
-  let editIndividualsButton = null
-  if (user.isPm || (project.hasCaseReview && project.canEdit)) {
-    editIndividualsButton = <EditFamiliesAndIndividualsButton />
-  } else if (project.canEdit) {
-    editIndividualsButton = <EditIndividualMetadataButton />
-  }
+  return datasetSections.map((sectionProps, i) =>
+    <DetailSection
+      {...sectionProps}
+      button={(datasetSections.length - 1 === i) ? <EditDatasetsButton user={user} /> : null}
+    />,
+  )
+})
 
-  const mmeSubmissionCount = mmeSubmissions.length
-  const deletedSubmissionCount = mmeSubmissions.filter(({ deletedDate }) => deletedDate).length
+Dataset.propTypes = {
+  project: PropTypes.object.isRequired,
+  samplesByType: PropTypes.object.isRequired,
+  user: PropTypes.object.isRequired,
+}
 
-  return (
```

Comment:
I broke this single big component that is connected to the state needed to render everything inside it into smaller helper components that only connect to the state they are actually using. This makes it more modular, and means that only the sub-components that actually need to re-render do so when the state changes, instead of rerendering the whole thing because part changed